sorter/src/test/java/sortpom/parameter/LineSeparatorParameterTest.java

```diff
@@ -18,46 +18,46 @@ public void lineSeparatorWithSomehtingElseShouldThrowException() throws Exceptio
 
         new PluginParametersBuilder()
                 .setEncoding("UTF-8")
-                .setFormatting("***", false, false)
+                .setFormatting("***", false, false, false)
```

Comment:
I follow others parameters being false or true, depending on tests

But you are right, now that this parameter is out of the setFormatting method, these changes are gone.

And do not worry, I am the same where I am the reviewer :)